app/controllers/lookup_keys_controller.rb

```diff
@@ -34,4 +33,8 @@ def destroy
   def controller_permission
     'external_variables'
   end
+
+  def resource
```

Comment:
don't we have anything similar in base controller? is this a reuse from the api controller? maybe consider to extract to a mixin?